src/VCR/LibraryHooks/CurlHook.php

```diff
@@ -154,6 +154,13 @@ public static function __callStatic($method, $args)
         return \call_user_func_array(array(__CLASS__, $localMethod), $args);
     }
 
+    public static function curlReset($curlHandle)
```

Comment:
For this there should be a test in [this class](https://github.com/php-vcr/php-vcr/blob/master/tests/VCR/LibraryHooks/CurlHookTest.php) as well.